packages/augur-tools/src/flash/scripts.ts

```
@@ -878,7 +942,12 @@ export function addScripts(flash: FlashSession) {
           displayShares: new BigNumber(0),
           expirationTime,
         };
-        result = fillOrder ? await user.augur.placeTrade(params) : await user.placeZeroXOrder(params)
+
+        try {
+          result = fillOrder ? await user.augur.placeTrade(params) : await user.placeZeroXOrder(params)
+        } catch(e) {
+          this.log(e);
```

Can you add a comment explaining why we're eating errors here?